File: v1/src/main/java/com/api/v1/dtos/RentResponseDto.java
package com.api.v1.dtos;

import java.math.BigDecimal;
import java.math.BigInteger;

public record RentResponseDto(
        BigInteger orderNumber,
        CustomerResponseDto customer,
        CarResponseDto car,
        PaymentResponseDto payment,
        int days,
        BigDecimal price,
        String returningDate,
        String returnedAt,
        String rentedAt
) {
}

File: v2/src/main/kotlin/com/api/v2/rent/rent/RentResponseDto.kt
package com.api.v2.rent.rent

import com.api.v2.car.domain.Car
import com.api.v2.customer.domain.Customer
import com.api.v2.customer.dtos.CustomerResponseDto
import com.api.v2.payment.domain.Payment
import java.math.BigInteger

data class RentResponseDto(
    val orderNumber: BigInteger,
    val customer: CustomerResponseDto,
    val car: Car,
    val payment: Payment,
    val rentedAt: String
)
